lib/ch/row_binary.ex

```diff
@@ -348,6 +348,16 @@ defmodule Ch.RowBinary do
     decode_type(rest)
   end
 
+  defp decode_type("AggregateFunction(" <> rest) do
+    [_agg_fun, rest] = :binary.split(rest, [", ", ")"], [:global, :trim])
+    decode_type(rest)
+  end
```

Comment:
it seems like this clause is not tested.

Reply:
Oh it seems the type returned for the `AggregateFunction` is just `DateTime('UTC')`.

Makes me wonder why it returns `SimpleAggregateFunction(min, Float64)` for the other

Reply:
Would it be OK to remove it for now until we have a use-case / a query for it? I saw that no other adapter supports `AggregateFunction(fun, [types])` type so I guess there might be something tricky to decoding it.

Reply:
- https://github.com/mymarilyn/clickhouse-driver
- https://github.com/ClickHouse/clickhouse-java
- https://github.com/mailru/go-clickhouse

these only support `SimpleAggregateFunction`, others, even the official `ch-go`, don't even seem to support `SimpleAggregateFunction` for some reason

Reply:
yeah that makes sense, I removed `AggregateFunction`